content/docs/thinking-in-react.md

```diff
@@ -1,24 +1,24 @@
 ---
 id: thinking-in-react
-title: Thinking in React
+title: Myślenie reaktowe
 permalink: docs/thinking-in-react.html
 redirect_from:
   - 'blog/2013/11/05/thinking-in-react.html'
   - 'docs/thinking-in-react-zh-CN.html'
 prev: composition-vs-inheritance.html
 ---
 
-React is, in our opinion, the premier way to build big, fast Web apps with JavaScript. It has scaled very well for us at Facebook and Instagram.
+Naszym zdaniem React dostarcza pierwszorzędnych narzędzi do budowy dużych i szybkich aplikacji webowych. Znakomicie sprawdza się na przykład w naszych zastosowaniach na Facebooku i w Instagramie.
```

Comment:
zastanawia mnie, dlaczego `na` Facebooku, ale `w` Instagramie? Trochę brak konsekwencji, chyba, że jest to uzasadnione w jakiś sposób?

Reply:
Swoją drogą słuszna uwaga, zazwyczaj słyszy się że coś jest `na` Facebooku. Ale jest to chyba bardziej potoczne stwierdzenie. W tym kontekście sugeruje użycie `w`.

Reply:
[Poradnia PWN](https://sjp.pwn.pl/poradnia/haslo/Zakupy-na-www-pwn-pl-czy-zakupy-w-www-pwn-pl;16673.html) mówi, że o stronach internetowych powinno się pisać "na Facebooku", a nie "w Facebooku".

Reply:
@iamarek Wiódł mnie  instynkt native speakera ;-) 

Moim zdaniem "**na** Facebooku" bardzo mocno utarło się w języku polskim. Utarło się wraz z samym Facebookiem tak mocno, że "**w** Facebooku" bolałoby w uszach. Natomiast Instagram moim zdaniem nie zaistniał w świadomości narodu w stopniu równym facebookowemu. Zaś partykuła "gram" w nazwie wciąż pobrzmiewa narzędziem jakimś, aplikacją. Stąd instynkt podpowiada mi "**w** Instagramie" - w narzędziu, w aplikacji, w firmie za tym stojącej (Facebooku).  Być może mylę się. Zagłosujmy?